feature/gribi/otg_tests/gribi_flush/README.md

```diff
@@ -0,0 +1,165 @@
+TE-18.4 - gRIBI Flush operation with Live traffic
+
+## Summary
+This test verifies when gRIBI flush operation is issued all the gRIBI rules are removed leaving the static routes intact.
+Ongoing traffic flows using gRIBI rules will switch to static route.
+
+## Topology
+
+* [`featureprofiles/topologies/atedut_2.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+
+## Test setup
+
+TODO: Complete test environment setup steps
```

Comment:
Please go ahead and complete this section to fulfill all the variables needed and remove the TODO.  Or if you want to use the exact same test setup as TE-18.1, then you could reference it